File: app/src/main/java/nl/rsdt/japp/application/activities/IntroActivity.java
package nl.rsdt.japp.application.activities;

import android.content.Intent;
import android.graphics.Color;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;

import com.github.paolorotolo.appintro.AppIntro;
import com.github.paolorotolo.appintro.AppIntroFragment;

import nl.rsdt.japp.R;

/**
 * @author Dingenis Sieger Sinke
 * @version 1.0
 * @since 8-7-2016
 * Description...
 */
public class IntroActivity extends AppIntro {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        addSlide(AppIntroFragment.newInstance("Welkom", "Welkom bij de Jotihunt app voor de RP", R.drawable.rp_logo_500x500, Color.parseColor("#993333")));
        addSlide(AppIntroFragment.newInstance("Kaart", "Bekijk de vossen, scouting-groepen, foto-opdrachten en de andere hunters op de kaart", R.drawable.ic_menu_camera, Color.parseColor("#66ccff")));
        addSlide(AppIntroFragment.newInstance("Waarnemen & melden", "Spot of hunt een vos via de app", R.drawable.ic_menu_camera, Color.parseColor("#99cc00")));
        addSlide(AppIntroFragment.newInstance("Zoeken", "Zoek een Scoutinggroeg op of een andere hunter", R.drawable.ic_menu_camera, Color.parseColor("#99cc00")));
        addSlide(AppIntroFragment.newInstance("Automatische Updates", "Wanneer de app actief is wordt alles automatisch bijgewerkt! Verander voorkeuren in de Instellingen onder updates", R.drawable.ic_menu_camera, Color.parseColor("#99cc00")));

        // OPTIONAL METHODS
        // Override bar/separator color.
        setBarColor(Color.parseColor("#996633"));

        setFadeAnimation();

        // Hide Skip/Done button.
        showSkipButton(true);
        setProgressButtonEnabled(true);

        // Turn vibration on and set intensity.
        // NOTE: you will probably need to ask VIBRATE permisssion in Manifest.
        setVibrate(true);
        setVibrateIntensity(30);
    }

    @Override
    public void onSkipPressed(Fragment currentFragment) {
        super.onSkipPressed(currentFragment);
        // Do something when users tap on Skip button.


        Intent intent = new Intent(this, MainActivity.class);
        startActivity(intent);
        finish();

    }

    @Override
    public void onDonePressed(Fragment currentFragment) {
        super.onDonePressed(currentFragment);

        Intent intent = new Intent(this, MainActivity.class);
        startActivity(intent);
        finish();

    }

    @Override
    public void onSlideChanged(@Nullable Fragment oldFragment, @Nullable Fragment newFragment) {
        super.onSlideChanged(oldFragment, newFragment);
        // Do something when the slide changes.
    }



}

File: app/src/main/java/nl/rsdt/japp/application/activities/IntroActivity.kt
package nl.rsdt.japp.application.activities

import android.content.Intent
import android.graphics.Color
import android.os.Bundle
import androidx.fragment.app.Fragment

import com.github.paolorotolo.appintro.AppIntro
import com.github.paolorotolo.appintro.AppIntroFragment

import nl.rsdt.japp.R

/**
 * @author Dingenis Sieger Sinke
 * @version 1.0
 * @since 8-7-2016
 * Description...
 */
class IntroActivity : AppIntro() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        addSlide(AppIntroFragment.newInstance(getString(R.string.welkom), getString(R.string.welkom_sub), R.drawable.rp_logo_500x500, Color.parseColor("#993333")))
        addSlide(AppIntroFragment.newInstance(getString(R.string.kaart), getString(R.string.kaart_desc), R.drawable.map_example, Color.parseColor("#66ccff")))
        addSlide(AppIntroFragment.newInstance(getString(R.string.waarnemen), getString(R.string.waarnemen_desc), R.drawable.spot_example, Color.parseColor("#99cc00")))
        addSlide(AppIntroFragment.newInstance(getString(R.string.auto_updates), getString(R.string.auto_updates_desc), R.drawable.ic_action_refresh, Color.parseColor("#993399")))

        // OPTIONAL METHODS
        // Override bar/separator color.
        setBarColor(Color.parseColor("#996633"))

        setFadeAnimation()

        // Hide Skip/Done button.
        showSkipButton(true)
        isProgressButtonEnabled = true

        // Turn vibration on and set intensity.
        // NOTE: you will probably need to ask VIBRATE permisssion in Manifest.
        setVibrate(true)
        setVibrateIntensity(30)
    }

    override fun onSkipPressed(currentFragment: Fragment?) {
        super.onSkipPressed(currentFragment)
        // Do something when users tap on Skip button.


        val intent = Intent(this, MainActivity::class.java)
        startActivity(intent)
        finish()

    }

    override fun onDonePressed(currentFragment: Fragment?) {
        super.onDonePressed(currentFragment)

        val intent = Intent(this, MainActivity::class.java)
        startActivity(intent)
        finish()

    }


}
